firefox-ios/Client/Frontend/Home/TopSites/DataManagement/UnifiedAds/UnifiedAdsCallbackTelemetry.swift

```diff
@@ -16,9 +16,7 @@ class DefaultUnifiedAdsCallbackTelemetry: UnifiedAdsCallbackTelemetry {
     private var logger: Logger
 
     init(
-        networking: ContileNetworking = DefaultContileNetwork(
-            with: makeURLSession(userAgent: UserAgent.mobileUserAgent(),
-                                 configuration: URLSessionConfiguration.defaultMPTCP)),
+        networking: ContileNetworking = DefaultContileNetwork(with: NetworkUtils.defaultURLSession()),
```

Comment:
Do we need to cache data ? `NetworkUtils.defaultURLSession` is now ephemeral where as `defaultMPTCP` wasn't. If we don't need cache then it should fine.

Reply:
Good catch! I missed that 😄 pushed a change to put that back to default